dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -305,11 +326,18 @@
 					&lpgpdma0 9>;
 			dma-names = "tx", "rx";
 			power-domain = <&io0_domain>;
+			ssp-index = <3>;
 			status = "okay";
+
+			ssp30: ssp@30 {
+				compatible = "intel,ssp-dai";
+				status = "okay";
+				reg = <0x30>;
+			};
 		};
 
 		ssp4: ssp@2c000 {
-			compatible = "intel,ssp-dai";
+			compatible = "intel,ssp";
```

Comment:
This will conflict with https://github.com/zephyrproject-rtos/zephyr/pull/71551 , but I'm ok to push this first if ready, easy to adjust.

Reply:
This PR is ready, but your pull request has more approvers :) I could update this pull request to reduce the possible number of SSP instances.

Reply:
Rebased